beard/similarity/tests/test_pairs.py

```diff
@@ -87,3 +89,22 @@ def test_absolute_difference():
 
     Xt = AbsoluteDifference().fit_transform(sp.csr_matrix(X))
     assert_array_almost_equal(Xt, [[0, 0], [1, 1], [0, 0], [1, 1]])
+
+
+def test_JaccardSimilarity():
+    X = np.array([[0, 1, 0, 1, 0, 1, 0, 1],
+                  [0, 0, 1, 1, 0, 1, 0, 1],
+                  [0, 1, 0, 1, 0, 0, 1, 0],
+                  [1, 0, 1, 1, 1, 1, 0, 1]])
```

Comment:
Tests dont pass if you add a row `[0, 0, 0, 0, 0, 0, 0, 0]` and the corresponding `[0.]` output.